recipes/lhapdf/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#This has to be here because autotools doesn't find the right cython
+(cd wrappers/python && cython --cplus lhapdf.pyx)
+./configure --prefix=$PREFIX
+make -j
+make check
+make install
+
+cd wrappers
+make
+make install
```

Comment:
Sorry I should've have been more specific with my comment above but do we have tests/checks for all the make calls?

Reply:
I believe the original `make check` takes care of that. See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=492947&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1398